src/daffodilDebugger.ts

```diff
@@ -6,13 +6,17 @@ import * as child_process from 'child_process';
 import { HttpClient } from 'typed-rest-client/HttpClient';
 
 // Function for getting the da-podil debugger
-export async function getDebugger() {
+export async function getDebugger(config: vscode.DebugConfiguration) {
+    let debugOutput = vscode.window.createOutputChannel("Debug");
+    debugOutput.appendLine("Testing")
+    
     let dapodilDebuggerVersion = await vscode.window.showInputBox({'prompt': "Enter in desired dapodil debugger version:"});
```

Comment:
Dont prompt for a version if the `startDapServer` flag is false

Reply:
removed those lines as I was testing something and forgot to delete them. Also made sure that if `disableDapServer` is set to true that the user will not be prompted for a version